.github/workflows/test.yml

```diff
@@ -6,12 +6,19 @@ on:
   pull_request:
     types: [opened, synchronize, reopened]
 jobs:
+  read-node-version:
+    name: "Read NodeJS version"
+    runs-on: ubuntu-22.04
+    steps:
+      - name: "Read .nvmrc"
+        run: "echo NVMRC=`cat .nvmrc` >> $GITHUB_ENV"
   lint:
     name: Lint
+    needs: read-node-version
     runs-on: ubuntu-22.04
     strategy:
       matrix:
-        node-version: [16.x]
+        node-version: [20.x]
```

Comment:
This can be removed since we use a single version of node specified in the nvmrc.

Reply:
Pushed a commit resolving this :+1: